cosmos/ununifi-beta-v1.json

```diff
@@ -0,0 +1,56 @@
+{
+  "rpc": "http://a.lcd.ununifi.cauchye.net:26657",
+  "rest": "https://a.lcd.ununifi.cauchye.net:1318",
+  "nodeProvider": {
+    "name": "UnUniFi",
+    "email": "info@ununifi.io",
+    "website":"https://ununifi.io/"
+  },
+  "chainId": "ununifi-beta-v1",
+  "chainName": "UnUniFi",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ununifi-beta-v1/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "GUU",
+    "coinMinimalDenom": "uguu",
+    "coinDecimals": 6,
+    "coinGeckoId": "ununifi",
```

Comment:
@Senna46 Could you remove `coinGeckoId` field?

Reply:
Removed. I forgot.